cmd/queue/main.go

```diff
@@ -342,6 +353,48 @@ func pushRequestLogHandler(currentHandler http.Handler) http.Handler {
 	return handler
 }
 
+func pushRequestMetricHandler(currentHandler http.Handler) http.Handler {
+	backend := os.Getenv("SERVING_REQUEST_METRICS_BACKEND")
```

Comment:
could this be initialed in `initEnv()` ?

Reply:
it's optional and only used in this function. I perfer to use it as local var.